api/v1alpha1/shared_types.go

```diff
@@ -20,15 +20,15 @@ const (
 	// DefaultDeploymentMemoryResourceRequests for deployment memory resource
 	DefaultDeploymentMemoryResourceRequests = "512Mi"
 	// DefaultEnvoyProxyImage is the default image used by envoyproxy
-	DefaultEnvoyProxyImage = "envoyproxy/envoy:distroless-dev"
+	DefaultEnvoyProxyImage = "envoyproxy/envoy:distroless-v1.29-latest"
```

Comment:
Will distroless-v1.29-latest changes when 1.29.2 comes out? Should we use distroless-v1.29.1 or distroless-v1.29-latest ?

Reply:
This logic is same as how we pin envoyproxy in v0.6 release.

Reply:
https://github.com/envoyproxy/gateway/pull/2098